crmsh/bootstrap.py

```diff
@@ -1828,6 +1832,34 @@ def init_admin():
     wait_for_resource("Configuring virtual IP ({})".format(adminaddr), "admin-ip")
 
 
+def evaluate_qdevice_quorum_effect(mode):
+    """
+    While adding/removing qdevice, get current expected votes and actual total votes,
+    to calculate after adding/removing qdevice, whether cluster has quorum
+    return different policy
+    """
+    quorum_votes_dict = utils.get_quorum_votes_dict()
+    expected_votes = int(quorum_votes_dict["Expected"])
+    actual_votes = int(quorum_votes_dict["Total"])
+    if mode == "add":
```

Comment:
This `add`, `remove` string look really week. Use constants instead

Reply:
Changed